keyboards/handwired/orbweaver/readme.md

```diff
@@ -0,0 +1,40 @@
+# orbweaver
+
+This firmware is for a Razer Orbweaver Chroma keypad, modded to replace the stock MCU
+with an Elite Pi RP2040 microcontroller. Mod instructions can be found on geekhack
+https://geekhack.org/index.php?topic=119396.0
+
+The firmware controls a 6x5 key matrix (4x5 keypad + 6 additional keys
+on the thumb pad and up to 4 additional keys added by user). It also controls a
+IS31FL3731 RGB Matrix controller (on the Chroma model) via I2C interface. This requires
+a single I2C Driver with 20 common anode RGB LEDs. I found that the I2C clock speed must
+be slowed down to 100 kHz for the IS31 chip to keep up (the default for RP2040 seems to
+be 400 kHz unless specified otherwise).  
+
+The default key map (layer 0) is a generic layer based on the original Razer defaults. 
+You can change or add layers as needed, and change static colors in keymap.c
+
+Many thanks to bomtarnes, Drashna, Dasky, sigprof, and casuanoob for helpful discussions
+and coding suggestions and snippets.
+
+* Keyboard Maintainer: [a_marmot](https://github.com/a-marmot)
+* Hardware Supported: RP2040 + IS31FL3731
+* Hardware Availability: https://geekhack.org/index.php?topic=119396.0
+
+Make example for this keyboard (after setting up your build environment):
+
+    qmk compile -kb /handwired/orbweaver -km default
+
+Flashing example for this keyboard:
+
+    When placed in bootloader mode, the RP2040 appears as an external flash drive. Just 
+    copy the .uf2 file to the mounted drive.
```

Comment:
```suggestion
with an Elite Pi RP2040 microcontroller. Mod instructions can be found on [geekhack](https://geekhack.org/index.php?topic=119396.0).

The firmware controls a 6x5 key matrix (4x5 keypad + 6 additional keys
on the thumb pad and up to 4 additional keys added by user). It also controls a
IS31FL3731 RGB Matrix controller (on the Chroma model) via I2C interface. This requires
a single I2C Driver with 20 common anode RGB LEDs. I found that the I2C clock speed must
be slowed down to 100 kHz for the IS31 chip to keep up (the default for RP2040 seems to
be 400 kHz unless specified otherwise).  

The default key map (layer 0) is a generic layer based on the original Razer defaults. 
You can change or add layers as needed, and change static colors in keymap.c

Many thanks to bomtarnes, Drashna, Dasky, sigprof, and casuanoob for helpful discussions
and coding suggestions and snippets.

* Keyboard Maintainer: [a_marmot](https://github.com/a-marmot)
* Hardware Supported: RP2040 + IS31FL3731
* Hardware Availability: [https://geekhack.org/index.php?topic=119396.0](https://geekhack.org/index.php?topic=119396.0)

Make example for this keyboard (after setting up your build environment):

    make handwired/orbweaver:default

Flashing example for this keyboard:

    make handwired/orbweaver:default:flash
    
```

Reply:
This command fails to compile on my system. 
"make: *** No rule to make target `handwired/orbweaver:default'.  Stop."

Also, I have not used this command to flash the RP2040. I have copied the uf2 file to the mounted drive, as described in current version of the readme.

Reply:
https://docs.qmk.fm/#/newbs_building_firmware
https://docs.qmk.fm/#/flashing?id=raspberry-pi-rp2040-uf2

Reply:
There might be better procedures, but anyone doing this mod will likely be using QMK for the first time (like me), so the code and readme instructions should probably closely follow qmk noob docs.